src/mmw/js/src/modeling/models.js

```diff
@@ -486,7 +490,9 @@ var ProjectModel = Backbone.Model.extend({
         return $.ajax({
             url: '/export/hydroshare?project=' + self.id,
             type: 'DELETE',
-        }).fail(function() {
+        }).success(function() {
```

Comment:
Prefer `.done` to `.success` for stylistic consistency, and because [`.success` is deprecated](http://api.jquery.com/jQuery.ajax/).

Reply:
Woops